<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RiskGui_Catchment_Scenario_Table_Wid</class>
 <widget class="QWidget" name="RiskGui_Catchment_Scenario_Table_Wid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_sort_prob">
       <property name="text">
        <string>Sort probability [-]</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_sort_prob_hyd">
       <property name="text">
        <string>Sort probability with HYD [1/a]</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_pre_risk">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Sort by predicted risk [%]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_pre_risk">
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>ECN</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ECO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>POP affected</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>POP endangered</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PYS</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
